deployment/README.md

```diff
@@ -63,83 +102,115 @@ The deployer must run under a service account defined as follows:
+
+    oc edit scc/privileged
+
+Add one line as the user at the end: (note, change `:logging:` below to the project of your choice)
+
+    - system:serviceaccount:logging:aggregated-logging-fluentd
+
+Give the account access to read labels from all pods (again with the correct project):
+
+    openshift admin policy add-cluster-role-to-user cluster-reader system:serviceaccount:logging:aggregated-logging-fluentd
+
 ## Run the Deployer
 
-You will need to specify the hostname at which Kibana should be exposed to client
-browsers, and also the master URL where client browsers will be directed for
-authenticating to OpenShift. These and other parameters are availalble:
+You will need to specify the hostname at which Kibana should be
+exposed to client browsers, and also the master URL where client
+browsers will be directed for authenticating to OpenShift. You should
+read the [ElasticSearch](#elasticsearch) section below before choosing
+ElasticSearch parameters for the deployer. These and other parameters
+are available:
 
 * `KIBANA_HOSTNAME` (required): External hostname where web clients will reach Kibana
 * `PUBLIC_MASTER_URL` (required): External URL for the master, for OAuth purposes
-* `IMAGE_PREFIX`: Specify prefix for logging component images; e.g. for "openshift/origin-logging-deployer:v1.1", set prefix "openshift/origin-"
-* `IMAGE_VERSION`: Specify version for logging component images; e.g. for "openshift/origin-logging-deployer:v1.1", set version "v1.1"
-* `ES_INSTANCE_RAM`: Amount of RAM to reserve per ElasticSearch instance.
-* `ES_CLUSTER_SIZE`: How many instances of ElasticSearch to deploy. At least 3 are needed for redundancy, and more can be used for scaling.
-* `ENABLE_OPS_CLUSTER`: If "true", configure a second ES cluster and Kibana for ops logs.
+* `IMAGE_PREFIX`: Specify prefix for logging component images; e.g. for "docker.io/openshift/origin-logging-deployer:v1.1", set prefix "docker.io/openshift/origin-"
+* `IMAGE_VERSION`: Specify version for logging component images; e.g. for "docker.io/openshift/origin-logging-deployer:v1.1", set version "v1.1"
+* `ES_INSTANCE_RAM`: Amount of RAM to reserve per ElasticSearch instance (e.g. 1024M, 2G). Defaults to 8GB; must be at least 512M (Ref.: [ElasticSearch documentation](https://www.elastic.co/guide/en/elasticsearch/guide/current/hardware.html#_memory).
+* `ES_CLUSTER_SIZE` (required): How many instances of ElasticSearch to deploy. At least 3 are needed for redundancy, and more can be used for scaling.
+* `ENABLE_OPS_CLUSTER`: If "true", configure a second ES cluster and Kibana for ops logs. (See [below](#ops-cluster) for details.)
 * `KIBANA_OPS_HOSTNAME`, `ES_OPS_INSTANCE_RAM`, `ES_OPS_CLUSTER_SIZE`: Parallel parameters for the ops log cluster.
 
-With example parameters:
+You run the deployer by instantiating a template. Here is an example with some parameters:
 
-    oc process logging-deployer-template \
+    oc process -n openshift logging-deployer-template \
```

Comment:
The `openshift` namespace is special. It's available to all users. Ansible should be uploading our templates there.